syft/serde.py

```diff
@@ -1238,6 +1244,16 @@ def _detail_object_wrapper(
     return obj_wrapper
 
 
+def _simplify_exception(e):
+    error_message = sys.exc_info()
+    return pickle.dumps(error_message)
+
+
+def _detail_exception(worker: AbstractWorker, error_tuple: tuple):
+    error = pickle.loads(error_tuple)
+    reraise(*error)
```

Comment:
Can we not just use raise for this?

Reply:
You need reraise to include the stacktrace, which is needed in the case you want to debug or if the error was unexpected and wasn't caught by the client